features/create_update_close.feature

```diff
@@ -0,0 +1,53 @@
+Feature: Case Management
+@Basic
+Scenario: Create a case, update a case, close a case	
+
+	Then I install the app at "28OQrA2"
+	Then I login with username "calabash" and password "auto"
+	Then I scroll until I see the "Start" text
+	Then I touch the "Start" text
+	Then I select case "Case Tests"
```

Comment:
@phillipm for some reason "I select case" works for modules and "I select module" works for cases when I try and run it...didn't want to rename and mess up other tests